templates/infra/linodeCluster.yaml

```diff
@@ -11,3 +11,7 @@ spec:
     apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
     kind: LinodeVPC
     name: ${VPC_NAME:=${CLUSTER_NAME}}
+  firewallRef:
```

Comment:
I think we will need a patch to the dns-based loadbal flavor to patch this out since it only will work for nodebalancers

Reply:
ah, good catch!

Reply:
Patched out the nb firewall out of dns-lb, cilium-bgp-lb, and dual-stack flavors